content/billing/tax-certificate.md

```diff
@@ -29,13 +29,16 @@ Palo Alto, CA 94306
 ## Register a tax certificate
 
 1. [Submit a Docker Support ticket](https://hub.docker.com/support/contact) to initiate the process to register a tax certificate.
-2. Enter the required information. Select **Billing** from the **Topic** drop-down.
-3. In the **Additional Information** field, list the Docker ID/namespace(s) of the accounts that you want to apply the tax exemption certificate to.  
+2. Enter the required information.
+3. Select **Billing** from the **Topic** drop-down.
+4. Select **Tax Information** from the **Subtopic** drop-down.
```

Comment:
Can we add query parameters to the contact url to pre-select these fields?

Reply:
Yeah something like that

Reply:
Done. Thank you.